src/main/java/games/strategy/engine/data/properties/ComboProperty.java

```diff
@@ -36,15 +33,19 @@ public ComboProperty(final String name, final String description, final T defaul
   private ComboProperty(final String name, final String description, final T defaultValue,
```

Comment:
Is this private constructor only called from the prior public constructor?  If so, then `allowNone` is always `false`.  That should clean up this constructor considerably.

Reply:
Good catch